### PR TITLE
implement Edric Storm using a new effect for dominance exclusion

### PR DESCRIPTION
**Pull request by zacchiro**
as part of this, Hodor implementation has been reworked to use the new effect
as well

### CODE REVIEW

File: server/game/player.js [hunk 1]
```diff
@@ -794,7 +794,7 @@ class Player extends Spectator {
 
     getDominance() {
         var cardStrength = this.cardsInPlay.reduce((memo, card) => {
-            if(!card.kneeled && card.getType() === 'character') {
+            if(!card.kneeled && card.getType() === 'character' && card.contributeToDominance) {
```
Comment (cryogen) [inline server/game/player.js]: super duper nitpick, but shoudl this be contributseToDominance, with a 's'?

File: server/game/cards/characters/02/hodor.js [hunk 2]
```diff
@@ -7,14 +7,10 @@ class Hodor extends DrawCard {
             match: this,
             effect: ability.effects.allowAsAttacker(false)
         });
-    }
```
Comment (cryogen) [inline server/game/cards/characters/02/hodor.js]: Very happy to see this disappear

File: server/game/cards/characters/05/edricstorm.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+const DrawCard = require('../../../drawcard.js');
+
+class EdricStorm extends DrawCard {
+
+    setupCardAbilities() {
+        this.reaction({
+            when: { onPhaseStarted: (event, phase) => phase === 'dominance' },
```
Comment (cryogen) [inline server/game/cards/characters/05/edricstorm.js]: I'm surprised that the linting doesnt complain about this!

File: server/game/player.js (hunk 1 above)
Comment (zacchiro) [inline server/game/player.js]: Fixed.

File: server/game/cards/characters/05/edricstorm.js (hunk 3 above)
Comment (zacchiro) [inline server/game/cards/characters/05/edricstorm.js]: Fixed.

**Comment by cryogen** [comment]
:+1:

**Pull request merged**